### ISSUE TITLE
[BUG] Indentation rulers in light theme are overly contrasting

### ISSUE THREAD

**Issue by marijnvdwerf** [decompme/decomp.me#1631]
**Describe the bug**
When the light theme is selected, the indentation markers are overly high contrasting with the background.

<img width="745" height="442" alt="Image" src="https://github.com/user-attachments/assets/e68f0d68-c2b0-4a00-b9e4-f4a0b32c9127" />

<img width="737" height="442" alt="Image" src="https://github.com/user-attachments/assets/4f96ed0c-c8fa-486a-a386-ede47bbc1495" />

**Potential solution:**
The indentation markers seem to be a hardcoded colour. Maybe they should be something like `color-mix(in srgb, var(--code-foreground), transparent 60%);`

**Comment by marijnvdwerf** [comment]
Fixed in #1632
